src/components/ButtonMenu.tsx

```diff
@@ -62,3 +85,19 @@ export type ImageMenuItemType = MenuItemBase & {
 export type MenuItem = MenuItemBase | IconMenuItemType | ImageMenuItemType;
 // This is done just to adapt to the React-Aria API for generating Sectioned lists of Menu Items.
 export type MenuSection = MenuItem & { items?: MenuItem[] };
+
+export function ContextualModal(props: ContextualModalProps) {
```

Comment:
**suggestion:** Would prefer this to be its own file. Maybe in `src/components/internal/ContextualModal.tsx`.

Reply:
got it. I noticed that when I created a file called `ContextualModal` Webstorm changes the file to a `text` file instead of `tsx`... if I rename the file to `ConTextualModal` it keeps the file as `tsx` but its kind of strange naming convention...